.expeditor/build.habitat.yml

```diff
@@ -0,0 +1,3 @@
+---
+origin: scotthain
```

Comment:
This is intentionally set to my origin while testing so I don't pollute core with artifacts.